ruby/lib/ci/queue/circuit_breaker.rb

```diff
@@ -50,25 +50,27 @@ def current_timestamp
         end
       end
 
-      def initialize(max_consecutive_failures:)
-        @max = max_consecutive_failures
-        @consecutive_failures = 0
-      end
+      class MaxConsecutiveFailures
```

Comment:
The MaxConsecutiveFailures circuit breaker was defined as the CircuitBreaker class. This cleans it up and puts it into the correct class.